nlmod/dims/layers.py

```diff
@@ -2269,6 +2266,50 @@ def get_max_overlap_model_layer(i):
     return modellayer
 
 
+def map_points_to_grid(x, y, ds=None, modelgrid=None):
+    """Map points to grid cells.
```

Comment:
I would add a note that this method is faster than using GridIntersect

Reply:
Done.